libasn1fix/asn1fix.c

```diff
@@ -74,6 +81,22 @@ asn1f_process(asn1p_t *asn, enum asn1f_flags flags,
 		}
 	}
 
+    if (flags & A1F_CASE_INSENSITIVE_FILENAMES) {
```

Comment:
NIT: Fix indenting for both cases

Reply:
> NIT: Fix indenting for both cases

My IDE likes to add spaces between 'if' and '(' even if I don't type a space ... Edited it to match the existing style with no space.